gapic-generator/templates/default/service/client/_client.erb

```diff
@@ -55,67 +71,51 @@ class <%= service.client_name %>
   #   `GRPC::Core::CallCredentials` object.
   #   A `Proc` will be used as an updater_proc for the Grpc channel. The proc
   #   transforms the metadata for requests, generally, to give OAuth credentials.
-  # @param scope [String, Array<String>]
-  #   The OAuth scope (or scopes) for this service. This parameter is ignored if
-  #   an updater_proc is supplied.
-  # @param timeout [Numeric]
-  #   The default timeout, in seconds, for calls made through this client.
-  # @param metadata [Hash]
-  #   Default metadata to be sent with each request. This can be overridden on a
-  #   per call basis.
   #
-  def initialize \
-      credentials: nil,
-      scope: nil,
-      timeout: DEFAULT_TIMEOUT,
-      metadata: nil,
-      lib_name: nil,
-      lib_version: nil
+  # @yield [config] Configure the <%= service.client_name %> client.
+  # @yieldparam config [<%= service.client_name %>::Configuration]
+  #
+  def initialize credentials: nil
```

Comment:
A question we're going to get is, why do credentials remain separate from the config?

Reply:
That's a good question. I don't know that I have a good answer. Would you be in favor of removing the argument completely?

Reply:
Yes, I'd like to explore getting rid of the argument completely and including credentials in the config. Not sure if this is easier said than done given the special logic and credentials class involved (although maybe some of that can be moved into the config class as well?) But it would be nice not to have credentials be a special case.

Reply:
@dazuma I've removed the `credentials` argument. PTAL.